contracts/FlowIDTableStaking.cdc

```diff
@@ -180,13 +179,17 @@ access(all) contract FlowIDTableStaking {
                 signatureAlgorithm: SignatureAlgorithm.BLS_BLS12_381
             )
 
+            // Verify the proof of possesion of the private staking key
+            assert(
+                stakeKey.verifyPoP(stakingKeyPoP.decodeHex()),
+                message: "Invalid Proof of Possesion for staking key"
```

Comment:
```suggestion
                message: 
                    "FlowIDTableStaking.NodeRecord.init: Cannot create node with ID "
                    .concat(id).concat(". The Proof of Possession (").concat(stakingKeyPoP)
                    .concat(") for the node's staking key (").concat(") is invalid")
```
We're trying to make the error messages more [descriptive and helpful](https://github.com/onflow/docs/issues/795) from now on, so we should start doing error messages like this in all our new code. I'll be going through the rest of the repo soon to update all the old ones